prac_05/wimbledon.py

```diff
@@ -0,0 +1,50 @@
+def read_wimbledon_data(filename):
+    """Read the Wimbledon champions data from a CSV file."""
+    champions_data = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        next(in_file)
+        for line in in_file:
+            data = line.strip().split(',')
+            year = data[0]
+            country = data[1]
+            champion = data[2]
+            champions_data.append((year, country, champion))
+    return champions_data
+
```

Comment:
In the case if the file does not exist or is unreadable for some reason, the program will raise an unhandled exception. You can improve this by adding error handling.